balancer/grpclb/grpclb_util.go

```diff
@@ -60,32 +61,41 @@ import (
 // so when grpclb client lose contact with remote balancers, the parent
 // ClientConn's resolver will re-resolve.
 type lbManualResolver struct {
-	scheme string
-	ccr    resolver.ClientConn
-
-	ccb balancer.ClientConn
-}
-
-func (r *lbManualResolver) Build(_ resolver.Target, cc resolver.ClientConn, _ resolver.BuildOptions) (resolver.Resolver, error) {
-	r.ccr = cc
-	return r, nil
-}
-
-func (r *lbManualResolver) Scheme() string {
-	return r.scheme
-}
+	// These fields are setup at creation time and are read-only after that, and
+	// therefore need not be protected with a mutex.
+	resolver.Builder                      // The underlying manual resolver builder.
+	resolver.Resolver                     // The underlying manual resolver.
```

Comment:
Can this just embed `*manual.Resolver` instead?

Reply:
We don't have this type anymore.